### ISSUE TITLE
#anchor links to references items are ungrouped for @see and @alias annotations

### ISSUE THREAD

**Issue by pete-hotchkiss** [SassDoc/sassdoc#464]
If I assign an `@see` or `@alias` within the comment block, and the item it's referencing is documented as being inside a `@group` then the Core doesn't seemingly return anything useable to the theme it can use to correctly construct the correct #group-type-name reference.

This occurs regardless of if I assign the `@group` annotation at a file or definition level.

Going on the default theme file /views/includes/annotations/see.html.swig, I'd perhaps expect to be able to do something like

```
<li><span class="item__cross-type">[{{ see.context.type }}]</span> <a href="#{{ see.group }}-{{ see.context.type }}-{{ see.context.name }}"><code>{{ see.context.name }}</code></a></li>
```

Unless of course I'm missing something obvious, and the JS that resolves the value object this annotation template uses does actually return such a property?

**Comment by pete-hotchkiss** [comment]
Worth adding - the similar annotation for ```@require``` has a ```require.item.group``` property which holds this - and those links work correctly.

**Comment by hyperbrave** [comment]
Am I correct in understanding that sassdoc is currently not able to @see $variables that are grouped?

**Comment by Camwyn** [comment]
+1
Using the default theme.
If I group my mixins, the links are ungrouped links. Even ungrouped items are in a group - undefined/General.

So if I have two mixins a & b and  b is in a group:
```
/// Mixin A
/// @see mixin-b

/// Mixin B
/// @group letters
/// @see mixin-a
```
I'll get two links:
```
file:///Users/camwyn/project/sassdoc/index.html#mixin-a
file:///Users/camwyn/project/sassdoc/index.html#mixin-b
```
Which should be
```
file:///Users/camwyn/project/sassdoc/undefined-index.html#mixin-a
file:///Users/camwyn/project/sassdoc/letters-index.html#mixin-b
```

**Comment by valeriangalliat** [comment]
Fixed as of sassdoc@2.7.4 and sassdoc-theme-default@2.8.5
